source/install-atlas-cli.txt

```diff
@@ -210,77 +210,126 @@ Follow These Steps
                .. tab:: MongoDB Community Edition
                   :tabid: mdb-comm
 
-                  Create the list file
-                  ``/etc/apt/sources.list.d/mongodb-org-5.0.list`` for
-                  your version of Ubuntu. Replace ``5.0`` with your
-                  edition of MongoDB.
+                  Select Ubuntu or Debian, then select your version.
 
                   .. tabs::
 
-                     .. tab:: Ubuntu 20.04 (Focal)
-                        :tabid: comm-focal
+                     .. tab:: Ubuntu
+                        :tabid: ubuntu
+
+                        Create the list file 
+                        ``/etc/apt/sources.list.d/mongodb-org-
+                        {+mdbVersion+}.list`` for your version of
```

Comment:
**Change**: This renders in the staging version with an extra space between `org-` and `6.0.list`.

Reply:
So most of the ones you mentioned below are intentional (or at least, that's how they've always been in the docs and they have worked in the past). This case is actually unintentional and is because I put a line break there, so thanks for catching!